lib/services/elasticsearch.js

```diff
@@ -315,7 +317,13 @@ function ElasticSearch (kuzzle, options, config) {
           if (result._source._kuzzle_info && !result._source._kuzzle_info.active) {
             // Replace the document if it is inactive
             return this.client.index(esRequest)
-              .then(res => refreshIndexIfNeeded.call(this, esRequest, _.extend(res, {_source: request.input.body})))
+              .then(res => {
```

Comment:
Or could we also reject an error if the user is found and is active (to avoid sending a request to ES that will finally throw an error) ?